CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.21- June 3, 2021
+## Breaking changes ⚠️
+* Align load style functions for MapboxMap and Snapshotter. ([#371](https://github.com/mapbox/mapbox-maps-android/pull/371))
```

Comment:
could we also suggest the specific API change, like:
```
Snapshotter.setUri(uri: String) -> Snapshotter.setStyleUri(styleUri: String)
Snapshotter.getUri()-> Snapshotter.getStyleUri()
Snapshotter.setJson(styleJson: String) -> Snapshotter.setStyleJson(styleJson: String)
Snapshotter.getJson()-> Snapshotter.getStyleJson()
MapboxMap.loadStyleJSON(styleJson: String) -> MapboxMap.loadStyleJson(styleJson: String)
``` 

But maybe this can be part of the release note instead of change log.

Reply:
Yes, let's put it in the release note.